src/3d/qgs3daxis.cpp

```diff
@@ -182,57 +181,35 @@ void Qgs3DAxis::onTouchedByRay( const Qt3DRender::QAbstractRayCaster::Hits &hits
     {
       if ( hits.at( mHitsFound ).entity() == mCubeRoot || hits.at( mHitsFound ).entity()->parent() == mCubeRoot )
       {
-#ifdef DEBUG
-        switch ( hits.at( mHitsFound ).primitiveIndex() / 2 )
-        {
-          case 0:
-            qDebug() << "East face";
-            break;
-
-          case 1:
-            qDebug() << "West face ";
-            break;
-
-          case 2:
-            qDebug() << "North face ";
-            break;
-
-          case 3:
-            qDebug() << "South face";
-            break;
-
-          case 4:
-            qDebug() << "Top face ";
-            break;
-
-          case 5:
-            qDebug() << "Bottom face ";
-            break;
-        }
-#endif
         switch ( hits.at( mHitsFound ).primitiveIndex() / 2 )
         {
           case 0: // "East face";
+            QgsDebugMsg( "Qgs3DAxis: East face clicked" );
```

Comment:
```suggestion
            QgsDebugMsgLevel( "Qgs3DAxis: East face clicked", 2 );
```

etc

(debug level 1 is reserved for errors only)